src/components/Input/Input.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './Input.scss';
+
+const Input = (props) => {
+  const {
+    className,
+    id,
+    isDisabled,
+    label,
+    onBlur,
+    onChange,
+    onFocus,
+    placeholder,
+    type,
+    value,
+  } = props;
+
+  const handleChange = (event) => {
```

Comment:
I am not seeing linter errors but we might need to dig into the config because those are missing semicolons and I'm pretty sure event gets flagged as a global due to MSIE. I will change these, and then figure out why my linter is not picking it up.

Reply:
I'm seeing the errors too.
<img width="347" alt="Screen Shot 2020-07-01 at 7 46 22 AM" src="https://user-images.githubusercontent.com/1447339/86257866-f6d25e80-bb6e-11ea-987f-100b9dc2ad0e.png">